sound/soc/intel/boards/bxt_pcm512x.c

```diff
@@ -28,74 +28,6 @@ struct bxt_card_private {
 	struct list_head hdmi_pcm_list;
 };
```

Comment:
@ranj063 So, basically we create a separate card for each type of DAI right? This looks good to me. Anyhow, I must admit I only superficially reviewed your patches. I will allocate some time to have a deeper look today.

Reply:
@dbaluta yes except that the hd audio codecs and dmics are clubbed into one.